js-exercises/rot-13/rot13.js

```diff
@@ -1,14 +1,14 @@
 function rot13(str) {
-  if(!str || typeof str != 'string') throw new Error('Invalid Parameter');
+  if (!str || typeof str != 'string') throw new Error('Invalid Parameter');
```

Comment:
Again, Better use {} even if one line :)
Makes it easy to read and good practise